### PR TITLE
docs: translate contribution guidelines en->zh

### LINKED ISSUE #6: `zh-cn/docs/contribution-guidelines/` need to be translated into Chinese version

### PR DESCRIPTION
**Pull request by niconical**
fix #6

### CODE REVIEW

File: content/zh-cn/docs/contribution-guidelines/_index.md [hunk 1]
```diff
@@ -5,25 +5,25 @@ description: >
...
-* Submitting bug report or feature request in [GitHub Issues](https://github.com/traas-stack/kapacity/issues)
-* Reviewing others code in [GitHub Pull Requests](https://github.com/traas-stack/kapacity/pulls)
-* Contributing to [documentations](https://github.com/traas-stack/kapacity-website)
+* 在 [GitHub Discussions](https://github.com/traas-stack/kapacity/discussions) 中回答问题
+* 在 [GitHub Issues](https://github.com/traas-stack/kapacity/issues) 中提交错误报告或新功能请求
+* Review [GitHub Pull Requests](https://github.com/traas-stack/kapacity/pulls) 中的他人代码
+* 为 [文档](https://github.com/traas-stack/kapacity-website) 做出贡献
 
-Here's the guideline for some contribution ways.
+这是一些贡献的指南。
```
Comment (zqzten) [inline content/zh-cn/docs/contribution-guidelines/_index.md]:
```suggestion
以下是一些具体的贡献指南。
```

File: content/zh-cn/docs/contribution-guidelines/_index.md [hunk 2]
```diff
@@ -5,25 +5,25 @@ description: >
...
 
-## Submitting issues
+## 提交问题
```
Comment (zqzten) [inline content/zh-cn/docs/contribution-guidelines/_index.md]:
```suggestion
## 提交 Issue
```

File: content/zh-cn/docs/contribution-guidelines/_index.md [hunk 3]
```diff
@@ -5,25 +5,25 @@ description: >
...
 
-Before submitting an issue, please check whether there has been similar existing issues and avoid duplicating them.
+在提交问题之前，请检查是否已经存在类似的问题，避免重复。
```
Comment (zqzten) [inline content/zh-cn/docs/contribution-guidelines/_index.md]:
```suggestion
在提交 Issue 之前，请检查是否已经存在类似的 Issue，避免重复。
```

File: content/zh-cn/docs/contribution-guidelines/_index.md [hunk 4]
```diff
@@ -5,25 +5,25 @@ description: >
...
 
-If you want to report a security vulnerability, make sure to choose the "Report a security vulnerability" option in "New issue". Do not submit it as a normal public issue.
+如果要报告安全漏洞，请务必在 `New issue` 中选择 `Report a security vulnerability` 选项。 不要将其作为普通的公共问题提交。
```
Comment (zqzten) [inline content/zh-cn/docs/contribution-guidelines/_index.md]:
```suggestion
如果要报告安全漏洞，请务必在 `New issue` 中选择 `Report a security vulnerability` 选项。不要将其作为普通的公开 Issue 提交。
```

**Review by zqzten — CHANGES_REQUESTED** [review]

**Comment by niconical** [comment]
Thanks, I've modified the corresponding translations based on your suggestions.

**Review by zqzten — APPROVED** [review]
LGTM, thanks! ❤️

**Pull request merged**